adrs/0011-annotations-lifespan.md

```diff
@@ -0,0 +1,28 @@
+# 10. Annotation types
+Date: 2022-07-05
```

Comment:
The date is weird, this decision was taken a long time ago but the ADR was written now. This is shady as doesn't match with the git annotations in the code.

Reply:
changed it to the last meaningful commit in the code 👌🏽

Reply:
changed it to the last meaningful commit in the code 👌🏽

Reply:
changed it to the last meaningful commit in the code 👌🏽